src/object.c

```diff
@@ -1647,7 +1647,7 @@ NULL
         addReplyVerbatim(c,report,sdslen(report),"txt");
         sdsfree(report);
     } else if (!strcasecmp(c->argv[1]->ptr,"purge") && c->argc == 2) {
-        if (jemalloc_purge() == 0)
+        if (zlibc_purge() == 0)
```

Comment:
this is not the only use of **our** `jemalloc_purge`.
it is also called in FLUSHALL etc.
ideally, what you should do is rename it to `zmalloc_purge` and add the non-jemalloc variants into it.

Reply:
I did it purposely to call it only in this context for now it might be less justified in other cases before some testing.

Reply:
in the past, this PURGE command was in DEBUG, so that might have been ok.
but with the current state, i think we want to properly test it before we merge it.